src/lib/core/datetime.c

```diff
@@ -395,3 +395,45 @@ datetime_increment_by(struct datetime *self, int direction,
 	self->nsec = nsec;
 	return 0;
 }
+
+bool
+datetime_datetime_sub(struct datetime_interval *res,
```

Comment:
Because in this case they were `left-hand-operand`_`right-hand-operand`_`operation`. Visually it looks better than `long-left-operand`_`op`_long-right-operand`. From my opinion.